core/capabilities/ccip/ccipnoop/crcwconfig.go

```diff
@@ -0,0 +1,21 @@
+package ccipnoop
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	ccipcommon "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/common"
+)
+
+// NoopChainRWProvider is a struct that implements the NoopChainRWProvider interface for Solana chains.
+type NoopChainRWProvider struct{}
+
+// GetChainWriter NoopChainRWProvider returns a new ContractWriter for Solana chains.
+func (g NoopChainRWProvider) GetChainWriter(ctx context.Context, pararms ccipcommon.ChainWriterProviderOpts) (types.ContractWriter, error) {
```

Comment:
nit: provider p, or unused _

```suggestion
func (p NoopChainRWProvider) GetChainWriter(ctx context.Context, pararms ccipcommon.ChainWriterProviderOpts) (types.ContractWriter, error) {
```